lib/index.d.ts

```diff
@@ -0,0 +1,170 @@
+import Vue from 'vue';
+import { Context, NuxtAppOptions } from '@nuxt/types';
```

Comment:
You shouldn't need this import (or the dependency on `@nuxt/types`)

Reply:
Right, removed.